src/tests/ftest/util/job_manager_utils.py

```diff
@@ -304,6 +358,15 @@ def assign_environment_default(self, env_vars):
         """
         self.export.update_default(env_vars.get_list())
 
+    def get_lib_path(self):
+        """Get path to mpi lib.
+
+        Returns: path to mpi lib
+        """
+        if not self.path:
+            raise CommandFailure("MPI path is not defined")
```

Comment:
Given the different paths we have in play here, might I suggest a slightly more clear error message:
```suggestion
            raise CommandFailure("Unable to obtain MPI library path: Full path for {} undefined!".format(self.command))
```